<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QWidget" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>779</width>
    <height>512</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>779</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_login_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>511</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 20pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>用户注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>确定密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;
border-radius:7px;
padding:2px 4px;
font: 10pt &quot;华文隶书&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;
border-radius:7px;
padding:2px 4px;
font: 10pt &quot;华文隶书&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sure_password">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;
border-radius:7px;
padding:2px 4px;
font: 10pt &quot;华文隶书&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请再次确认密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 127);
color: rgb(255, 255, 255);
font: 25 9pt &quot;Bahnschrift Light&quot;;
font: 10pt &quot;华文隶书&quot;;
border-radius:7px;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_sure">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>440</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 127);
color: rgb(255, 255, 255);
font: 25 9pt &quot;Bahnschrift Light&quot;;
font: 10pt &quot;华文隶书&quot;;
border-radius:7px;</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>350</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;
border-radius:7px;
padding:2px 4px;
font: 10pt &quot;华文隶书&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>输入手机号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;
border-radius:7px;
padding:2px 4px;
font: 10pt &quot;华文隶书&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>输入姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>手机号</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
